### ISSUE TITLE
Mutator unable to ignore mutation counts

### ISSUE THREAD

**Issue by suffdota** [VoidArkMods/QoLPlus#71]
Propagator has an ini option to ignore mutation counts, mutator does not.
After 20+/20 is reached on male, half of mutations on the mutator do not occur when the mutation is selected to come from the male parent.
Other half where mutation is selected to come from the female parent correctly adds a mutation as intended.

**Comment by Architoc** [comment]
Yes it would be great to have this kind of option also for the mutator to limit the mutations that we can obtain on the dinos
